acceptance/testdata/launcher/windows/container/layers/config/metadata.toml

```diff
@@ -1,3 +1,8 @@
+[[buildpacks]]
+  id = "0.5/buildpack"
+  version = "0.0.1"
+  api = "0.5"
```

Comment:
Don't we want to add it to buildpack api 0.6?
And if yes, is it possible to just use the latest api here?

Reply:
Ohh... that's interesting. This is a weird one because technically exec.d support was introduced in buildpack api 0.5 (according to the spec) but for Windows it's different. I am happy to change the fixture to avoid any confusion.